### PR TITLE
Prefer character classes to alternators in regex

### PR DESCRIPTION
**Pull request by varun8338**

### CODE REVIEW

File: oshi-core/src/main/java/oshi/hardware/platform/unix/freebsd/FreeBsdCentralProcessor.java [hunk 1]
```diff
@@ -149,7 +149,7 @@ protected Pair<List<LogicalProcessor>, List<PhysicalProcessor>> initProcessorCou
     }
 
     private static List<LogicalProcessor> parseTopology() {
-        String[] topology = BsdSysctlUtil.sysctl("kern.sched.topology_spec", "").split("\\n|\\r");
+        String[] topology = BsdSysctlUtil.sysctl("kern.sched.topology_spec", "").split("[\\n\\r]");
```
[the remark this answers is not in the document]
Comment (varun8338) [reply oshi-core/src/main/java/oshi/hardware/platform/unix/freebsd/FreeBsdCentralProcessor.java:152]: @sonatype-lift ignoreall

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 # 6.2.2 (in progress)
 
 * Your contribution here!
+* [#2097](https://github.com/oshi/oshi/pull/2097): Fixed Issue #2097, by updating regEx - [@varun83388](https://github.com/varun83388).
```
Comment (dbwiddis) [inline CHANGELOG.md:4]: This should be the number of your PR, 2108.

**Review by dbwiddis — COMMENTED** [review]
One small tweak please :)

File: CHANGELOG.md (hunk 2 above)
Comment (dbwiddis) [reply CHANGELOG.md:4]: Also I liked the more detailed description message in your earlier commit, that matched the issue title.

Comment (varun8338) [reply CHANGELOG.md]: I've made the changes, are you able to see it?

Comment (varun8338) [reply CHANGELOG.md]: > Also I liked the more detailed description message in your earlier commit, that matched the issue title.

Thank you for being patient, really motivates the first-timers! :)

Comment (dbwiddis) [reply CHANGELOG.md]: I can see the new number but still the old description message.  Referencing another issue doesn't really tell users what changed.  In your #2107 version you used "Prefer character classes to alternators in regex" which is good.

**Review by dbwiddis — CHANGES_REQUESTED** [review]
I think you saw my comment about changing the commit message/description (you added an emoji to it) but haven't done that yet?   

A change log message just referencing an issue # isn't very descriptive. Can you change that to how you had it in #2107, "Prefer character classes to alternators in regex" ?

**Review by dbwiddis — APPROVED** [review]
Looks great! I'll merge as soon as CI runs.

Thanks for your contribution and your patience.  Hope you'll continue to contribute to Open Source.

**Comment by varun8338** [comment]
> Looks great! I'll merge as soon as CI runs.
> 
> Thanks for your contribution and your patience. Hope you'll continue to contribute to Open Source.

**Pull request merged**